toolkits/global/packages/global-ratings-survey/scss/50-components/_ratings-survey.scss

```diff
@@ -0,0 +1,117 @@
+.c-ratings-survey {
+	padding: $global-ratings-survey-container-padding;
+	background-color: $global-ratings-survey-container-background-color;
+	overflow: hidden;
+	font-family: $global-ratings-survey-font-family;
+}
+
+.c-ratings-survey__container {
+	display: flex;
+	justify-content: center;
+}
+
+// !important used to ensure overrides heading styles on all of our imprint sites
+.c-ratings-survey__heading {
+	margin: 0 0 $global-ratings-survey-heading-margin !important;
+	font-family: inherit !important;
+	font-size: $global-ratings-survey-heading-font-size !important;
+	line-height: $global-ratings-survey-line-height !important;
+	font-weight: $global-ratings-survey-heading-font-weight !important;
+}
+
+.c-ratings-survey__submit-message div {
+	display: flex;
+	justify-content: flex-start;
+	align-items: center;
+}
+
+.c-ratings-survey__submit-message svg {
+	width: $global-ratings-survey-message-icon-size;
+	height: $global-ratings-survey-message-icon-size;
+	margin-right: $global-ratings-survey-message-icon-margin;
+}
+
+.c-ratings-survey__submit-message p {
+	margin: 0;
+	line-height: $global-ratings-survey-line-height;
+	font-family: inherit;
+	font-size: $global-ratings-survey-message-font-size;
+}
+
+.c-ratings-survey__button {
+	// TODO: Remove this mixin and replace with Brand Context button styles once work to improve them has been carried out
+	@include global-ratings-survey-button;
+}
+
+.c-ratings-survey a {
+	display: inline-block;
+	margin-top: $global-ratings-survey-link-margin;
+	font-family: inherit;
+	font-size: $global-ratings-survey-message-font-size;
+}
+
+.c-ratings-survey__visually-hidden,
+.c-ratings-survey legend.c-forms__label {
+	/* hide visually, not to assistive tech */
```

Comment:
does this do something different to using the [visually hide](https://github.com/springernature/frontend-toolkits/blob/main/context/brand-context/default/scss/30-mixins/hiding.scss#L16) mixin available in `brand-context`?

available by using `@include u-visually-hidden`

Reply:
Oooo, good catch. In which case this looks good, and yeah if you could add as an issue that would be wonderful 😃